examples/rich_structure/BUILD

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+licenses(["notice"])
+
+load("@rules_pkg//:mappings.bzl", "pkg_attributes", "pkg_filegroup", "pkg_files", "pkg_mkdirs", "strip_prefix")
+load("@rules_pkg//:pkg.bzl", "pkg_tar", "pkg_zip")
+
+pkg_files(
```

Comment:
If this is intended to be an exemplar, it might be worth adding in even more comments to, for example, explain the purpose behind things that might look odd or novel.

Much of this might be self-explanatory, but it may be worth providing more annotations in the likely event that people are going to this instead of reading the reference manual.

Reply:
I added some more. There is still more that we should add incrementally.